src/SFML.Graphics/VertexBuffer.cs

```diff
@@ -88,20 +91,24 @@ public VertexBuffer(VertexBuffer copy)
         ////////////////////////////////////////////////////////////
         /// <summary>
         /// OpenGL handle of the vertex buffer or 0 if not yet created
-        /// 
+        /// </summary>
+        /// <remarks>
         /// You shouldn't need to use this property, unless you have
         /// very specific stuff to implement that SFML doesn't support,
         /// or implement a temporary workaround until a bug is fixed.
-        /// </summary>
+        /// </remarks>
         ////////////////////////////////////////////////////////////
         public uint NativeHandle
         {
-            get { return sfVertexBuffer_getNativeHandle(CPointer); }
+            get {
+                #warning Use NativeHandle cautiously.  Most people shouldn't need it.
```

Comment:
Not sure about the warning here. Can one turn it off? It seems to get triggered on the build itself, see the annotations below from every build.

If you really want to keep it, remove the double space between the two sentences.